app/routes/organisation.js

```diff
@@ -1,14 +1,62 @@
-import AuthorizeRoute from './authorize';
+import AuthorizeRoute from "./authorize";
 
 export default AuthorizeRoute.extend({
   model(params) {
-    return this.store.peekRecord("gc_organisation", params.organisation_id) || 
-      this.store.findRecord("gc_organisation", params.organisation_id, { reload: true });
+    return (
+      this.store.peekRecord("gc_organisation", params.organisation_id) ||
+      this.store.findRecord("gc_organisation", params.organisation_id, {
+        reload: true
+      })
+    );
   },
 
-  setupController(controller, model){
+  async setupController(controller, model) {
     this._super(controller, model);
-    let gcOrganisationUsers = this.store.peekAll('organisationsUser').filterBy('organisationId', parseInt(model.id));
-    controller.set('gcOrganisationUsers', gcOrganisationUsers);
+    this.setOrganisationUsers(model);
+    this.setOrganisationTypes(model);
+    this.setCountry(model);
+  },
+
+  // set the selected country value
+  async setCountry(model) {
+    const country = model.get("country");
+    this.controller.set("country", {
+      id: country.get("id"),
+      nameEn: country.get("nameEn")
+    });
+
+    this.controller.set("countryValue", { country_id: country.get("id") });
+  },
+
+  // set organisation users
+  async setOrganisationUsers(model) {
+    let gcOrganisationUsers = this.store
+      .peekAll("organisationsUser")
+      .filterBy("organisationId", parseInt(model.id));
+    this.controller.set("gcOrganisationUsers", gcOrganisationUsers);
+  },
```

Comment:
I'm trying to understand where the organisationsUsers are used within this feature 🤔  
cc @bharat619

Reply:
yes @patrixr , you are right.
its not a part of this feature. I just moved the existing organisation-users related code to a separate method